nossomercadolivre-api/src/main/java/com/github/nossomercadolivre/User.java

```diff
@@ -0,0 +1,140 @@
+package com.github.nossomercadolivre;
+
+import com.github.nossomercadolivre.exception.InvalidPasswordException;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+import javax.persistence.*;
+import javax.validation.constraints.*;
+import java.io.Serializable;
+import java.time.Instant;
+import java.util.Objects;
+
+import static javax.persistence.GenerationType.IDENTITY;
+
+@Entity
+@Table
+public class User implements Serializable {
+
+    @Id
+    @GeneratedValue(strategy = IDENTITY)
+    private Long id;
+
+    @CreatedDate
```

Comment:
*Usamos tudo que conhecemos que está pronto. Só fazemos código do zero se for estritamente necessário.*

Pq precisa dessa annotation quando já podemos instanciar a nova data na inicialização do objeto? Inclusive você inicializou.

Reply:
Tem razão, vou deixar só a anotation :D e remover o setCreatedDate que está relacionado com um [ponto lá em baixo](https://github.com/netstart/nosso-mercado-livre/pull/2#discussion_r389468545).

Reply:
Pq a gente deixou a annotation?

Reply:
Bem, dado as opções
 - Remover anotation
 - Remover a data no construtor
Preferi remover a data no construtor, e deixar o framework instanciar a data para mim.

@asouza Preferiria que fosse o contrário?

Reply:
Preferia :) hehehe